docs/docs/behaviors/macros.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Macro Behavior
+sidebar_label: Macros
+---
+
+## Summary
+
+The macro behavior allows configuring a sequence of other behaviors to invoke
+when the macro is pressed, including support for completing part of the sequence
+once the macro is released.
+
+## Macro Definition
+
+Each macro you want to use in your keymap gets defined first, then bound in your keymap.
+
+A macro definition looks like:
+
+```
+/ {
+    behaviors {
+        alt_tab: alt_tab_macro {
+			label = "ALT_TAB_MAC";
+			compatible = "zmk,behavior-macro";
+			#binding-cells = <0>;
+			bindings
+				= <&macro_press &kp LALT>
+				, <&macro_tap>
+				, <&kp TAB>
+                , <&macro_pause_for_release>
+				, <&macro_release &kp LSHFT>
+				;
+		};
+    };
+};
+```
+
+The macro can then be bound in your keymap by referencing it by the label `&alt_tab`.
+
+### Bindings
+
+Like hold-taps, macros are created by composing other behaviors, and any of those behaviors can
+be added to the `bindings` list.
+
+### Wait Time
+
+The wait time setting controls how long of a delay is introduced between activations of each behavior in the macro bindings sequence.
+The default wait time is 100ms, but the value can be overridden by setting the `wait-ms` property on the macro, e.g.:
+
+```
+/ {
+    behaviors {
+        ab_macro: ab_macro {
+			label = "AB_MAC";
+			compatible = "zmk,behavior-macro";
+			#binding-cells = <0>;
+            wait-ms = <30>;
+			bindings
+				= <&kp A &kp B>
+				;
+		};
+    };
```

Comment:
There is a mixture of tabs vs spaces for indentation here (and in the other code blocks in this file).